docling/backend/msword_backend.py

```diff
@@ -952,34 +974,39 @@ def _handle_text_elements(  # noqa: C901
                 paragraph_elements=paragraph_elements,
             )
             for text, format, hyperlink in paragraph_elements:
-                doc.add_text(
+                t3 = doc.add_text(
                     label=DocItemLabel.PARAGRAPH,
```

Comment:
Can we use `TEXT` instead of `PARAGHRAPH`. I would like to deprecate `PARAGHRAPH`.